src/store/actions/Chat.ts

```diff
@@ -0,0 +1,161 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import firebase from "firebase/app";
+
+export const SEND_GLOBAL_CHAT: string = "SEND_GLOBAL_CHAT";
```

Comment:
Up for a discussion: 
```
export const GLOBAL_CHAT = "GLOBAL_CHAT";
```

Reply:
Good call out. My leaning would be toward including the verb so it's immediately obvious when reading the code, that this is a create operation. An action of type `GLOBAL_CHAT` could be interpreted by a reader as "get global chat" or as "hey, a global chat happened, and the UI needs to update".

I'm leaning towards keeping `SEND` in there. But happy to be convinced otherwise!